src/declarations/backend/backend.factory.certified.did.js

```diff
@@ -57,6 +49,14 @@ export const idlFactory = ({ IDL }) => {
 		version: IDL.Opt(IDL.Nat64),
 		enabled: IDL.Bool
 	});
+	const UserToken = IDL.Record({
+		decimals: IDL.Opt(IDL.Nat8),
+		version: IDL.Opt(IDL.Nat64),
+		enabled: IDL.Opt(IDL.Bool),
+		chain_id: IDL.Nat64,
+		contract_address: IDL.Text,
+		symbol: IDL.Opt(IDL.Text)
+	});
```

Comment:
I wonder why this moved.  It looks identical.  Have I missed something?

Reply:
I don't know why it moved but, we generate the did types and related javascript files automatically (`npm run generate`). So it's related to the Candid parser.